### PR TITLE
Added "Two bots chatting to each other"

### PR DESCRIPTION
**Pull request by andygrunwald**
A small discussion between two bots (@greenkeeperio-bot and @googlebot): https://github.com/ampproject/amphtml/pull/484#issuecomment-146279527

### CODE REVIEW

File: _data/entries.yml [hunk 1]
```diff
@@ -100,4 +100,8 @@
 
 - title: "Removed use of gendered pronoun"
   link: "https://github.com/joyent/libuv/pull/1015#issuecomment-29538615"
-  type: "pull request"
\ No newline at end of file
+  type: "pull request"
+
+- title: "Two bots chatting to each other"
+  link: "https://github.com/ampproject/amphtml/pull/484#issuecomment-146279527"
```
Comment (mehulkar) [inline _data/entries.yml]: can you link to the PR instead of the comment?

**Comment by mehulkar** [comment]
@andygrunwald this is awesome! definitely worthy, imo (do you know of any other examples of this on GH?). might be more appropriate to add the _first_ example of this, if that is known.

**Comment by andygrunwald** [comment]
@mehulkar I linked the PR instead of the comment.
I don`t know any other examples of this kind. Sorry.

But maybe we will get this in and if you, a github user or i discover an earlier discussion between bots, we will replace it. Does this make sense?

**Comment by andygrunwald** [comment]
Thank you :)

**Comment by mehulkar** [comment]
Thanks for the commit!

**Pull request merged**